### PR TITLE
docs(K8s): Add instructions for loading the examples

### PR DESCRIPTION
**Pull request by charris-msft**
This took me a long time to figure out because I'm new to Superset, Helm & Kubernetes.
I want to make it easier for others in the future.

### SUMMARY
Adding instructions for loading the examples for Kubernetes deployments as this is different from Docker deployments.
This took me a long time to figure out because I'm new to Superset, Helm & Kubernetes, so I want to make it easier for others in the future.

### TESTING INSTRUCTIONS
1. run `helm upgrade --install superset superset/superset`
Result: Examples will not be loaded and you will not see the `examples` database in Superset

2. Follow added instructions then run `helm upgrade --install --alues my_values.yaml superset superset/superset`
Result: Examples will be loaded and you will see the `examples` database in Superset

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by srinify — APPROVED** [review]

**Comment by rusackas** [comment]
Closing/re-opening to kick CI.

File: docs/docs/installation/running-on-kubernetes.mdx [hunk 1]
```diff
@@ -396,3 +396,12 @@ configOverrides:
...
 ```
+#### Load the Examples data and dashboards
+If you are trying Superset out and want some data and dashboards to explore, you can load some examples by creating a `my_values.yaml` and deploying it as described above in the **Configure your setting overrides** step of the **Running** section.
+To load the examples, add the following to the `my_values.yaml` file:
+```yaml
+init:
+  loadExamples: true
+```
+
+
```
Comment (rusackas) [inline docs/docs/installation/running-on-kubernetes.mdx:407]:
```suggestion
```

```

File: docs/docs/installation/running-on-kubernetes.mdx [hunk 2]
```diff
@@ -396,3 +396,9 @@ configOverrides:
         "--disable-extensions",
     ]
 ```
+#### Load the Examples data and dashboards
+If you are trying Superset out and want some data and dashboards to explore, you can load some examples by creating a `my_values.yaml` and deploying it as described above in the **Configure your setting overrides** step of the **Running** section.
+To load the examples, add the following to the `my_values.yaml` file:
+```yaml
+init:
+  loadExamples: true
```
Comment (rusackas) [inline docs/docs/installation/running-on-kubernetes.mdx:404]:
```suggestion
  loadExamples: true
  
```

Comment (rusackas) [reply docs/docs/installation/running-on-kubernetes.mdx:404]: Ugh... I think I overcorrected on my last move... I believe there needs to be ONE newline... not two, not zero :)

File: docs/docs/installation/running-on-kubernetes.mdx [hunk 3]
```diff
@@ -396,3 +396,10 @@ configOverrides:
...
+
```
Comment (rusackas) [inline docs/docs/installation/running-on-kubernetes.mdx:405]:
```suggestion
  ```
```

Comment (rusackas) [inline docs/docs/installation/running-on-kubernetes.mdx:405]:
```suggestion
```
```

Comment (rusackas) [inline docs/docs/installation/running-on-kubernetes.mdx:405]:
```suggestion
init:
  loadExamples: true
```

```

Comment (rusackas) [inline docs/docs/installation/running-on-kubernetes.mdx:405]:
````suggestion
```yaml
init:
  loadExamples: true
```

````

File: docs/docs/installation/running-on-kubernetes.mdx [hunk 4]
```diff
@@ -396,3 +396,11 @@ configOverrides:
...
     ]
 ```
+#### Load the Examples data and dashboards
+If you are trying Superset out and want some data and dashboards to explore, you can load some examples by creating a `my_values.yaml` and deploying it as described above in the **Configure your setting overrides** step of the **Running** section.
+To load the examples, add the following to the `my_values.yaml` file:
+```yaml
+init:
+  loadExamples: true
+```
+
```
Comment (rusackas) [inline docs/docs/installation/running-on-kubernetes.mdx:406]:
```suggestion
```

**Pull request merged**